README.md

```diff
@@ -32,30 +40,17 @@ Feedback, suggestions, questions or concerns? E-mail us at [support@parsely.com]
 
 Feedback, suggestions, questions or concerns? E-mail us at [support@parsely.com](mailto:support@parsely.com) -- we always want to hear from you.
 
-## Tests ##
-
-1. Follow the steps to install VirtualBox, Vagrant and `vagrant-hostsupdater` at [http://vccw.cc/](http://vccw.cc/).
-1. Navigate to the directory on your host machine to which you downloaded vccw.
-1. Within that directory, navigate to `www/wordpress/wp-content/plugins` and clone this repo -- `git clone https://github.com/Parsely/wp-parsely.git`
-1. Checkout the branch on which you've been working
-1. Navigate back up to the root of the `vccw` directory and run `vagrant up`
-1. Ssh into the Vagrant machine with `vagrant ssh`
-1. Delete extra wordpress files in /tmp with `rm -rf /tmp/wordpress*` (see this issue for context: https://github.com/wp-cli/wp-cli/issues/1938)
-1. Run the test init script, if you haven't already: `bash bin/install-wp-tests.sh wptest root <db password>`
-1. Navigate to `/var/www/wordpress/wp-content/plugins/wp-parsely` and run the test suite with the command `phpunit`
-
-
-## Frequently Asked Questions ##
+## Frequently Asked Questions
```

Comment:
While these instructions are quite out of date, do we want to add any suggested workflow here or just don't worry about it?

Reply:
At this point, I was just consolidating the readme's; other changes to the content can be done afterwards in a new PR, when there's only one file to edit.

Reply:
:+1: works for me